src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -2934,7 +2934,8 @@ public IEnumerable<ISymbol> GetSymbolsWithName(Func<string, bool> predicate, Sym
                 var result = new HashSet<ISymbol>();
                 var spine = new List<MergedNamespaceOrTypeDeclaration>();
 
-                AppendSymbolsWithName(spine, _compilation.Declarations.MergedRoot, predicate, filter, result, cancellationToken);
```

Comment:
`MergedRootDeclaration` is a new helper property on `Compilation` which calls into `DeclarationTable.GetMergedRoot` which is the method version of the previous `MergedRoot` property. The `Unordered` qualifier is no longer needed since `GetMergedRoot` returns the ordered collection.